tests/run.sh

```diff
@@ -5,17 +5,25 @@ RUN_S3=""
 function get_requirements() {
     # Download what we need.
     mkdir -p downloaded-tools
-    curl "https://d25phv8h0wbwru.cloudfront.net/${TEST_BRANCH}/tip/mender-artifact" \
+    local artifact_version=$(../extra/release_tool.py --version-of mender-artifact)
+
+    curl "https://d25phv8h0wbwru.cloudfront.net/${artifact_version}/tip/mender-artifact" \
          -o downloaded-tools/mender-artifact \
          -z downloaded-tools/mender-artifact
 
     chmod +x downloaded-tools/mender-artifact
-    export PATH=$PWD/downloaded-tools:$PATH
-
 
     curl "https://s3.amazonaws.com/mender/temp_${TEST_BRANCH}/core-image-full-cmdline-vexpress-qemu.ext4" \
          -o core-image-full-cmdline-vexpress-qemu.ext4 \
          -z core-image-full-cmdline-vexpress-qemu.ext4
+
+   curl "https://s3-eu-west-1.amazonaws.com/stress-client/release/mender-stress-test-client" \
+        -o downloaded-tools/mender-stress-test-client \
+        -z downloaded-tools/mender-stress-test-client
```

Comment:
We should not use downloaded tools in release builds, so we need to add this to the non-`get_requirements` code block as well (the `-n "$BUILDDIR"` block). I suppose we can simply clone the repository and build it, or?